Targets/AppUIKit/Sources/verifier/VerifyDLView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import SpruceIDMobileSdkRs
+
+struct VerifyDL: Hashable {}
+
+struct VerifyDLView: View {
+
+    @State var success: Bool?
+
+    @Binding var path: NavigationPath
+
+    var body: some View {
+        if success == nil {
+            ScanningComponent(
+                path: $path,
+                scanningParams: Scanning(
+                    subtitle: "Scan the\nback of your driver's license",
```

Comment:
can we get rid of the breakline and let Swift UI handle linebreak?

Reply:
this is just to make it equal to the designs